scripts/install/opt/seagate/cortx/motr/sanity/cortx_lnet_sanity.sh

```diff
@@ -158,7 +158,7 @@ is_sudoed
 workflow
 
 verify_lnet_installation
-die_if_failed $? "Required Lustre packages [${LNET_INSTALLED_COMPONENTS[@]}] were not found."
+die_if_failed $? "Required Lustre packages [${LNET_INSTALLED_COMPONENTS[*]}] were not found."
```

Comment:
Why is this needed?

Reply:
Hi did this because of the below issue.
`[root@ssc-vm-g4-rhev4-1908 sanity]# shellcheck cortx_lnet_sanity.sh

In cortx_lnet_sanity.sh line 161:
die_if_failed $? "Required Lustre packages [${LNET_INSTALLED_COMPONENTS[@]}] were not found."
                                            ^-- SC2145: Argument mixes string and array. Use * or separate argument.`